spring-cloud-starter-single-step-batch-job/pom.xml

```diff
@@ -9,6 +9,12 @@
 
 	<artifactId>spring-cloud-starter-single-step-batch-job</artifactId>
 
+	<properties>
+		<test.containers.version>1.9.1</test.containers.version>
+		<test.rabbit.containers.version>1.12.5</test.rabbit.containers.version>
```

Comment:
There is a newer version of testcontainers (and its rabbitmq module): 1.14.3. Any reason not using the latest version?

Reply:
Updated to latest version.